frontend/src/views/main/project/TestSuite.vue

```diff
@@ -102,6 +113,11 @@ const { suite, inputs, executions, hasTest, hasInput, statusFilter, searchFilter
 
 const openWorkerInstructions = ref(false);
 
+const suiteName = computed(() => {
+  if (!suite.value) return `Unnamed test suite`;
+  return suite.value.name ? suite.value.name : `Unnamed test suite`;
```

Comment:
In js you can use nullish coalescing instead of ternary in this case: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing 

It won't work if name is an empty string but logical or (`||`) would work

```suggestion
  return suite.value?.name ?? 'Unnamed test suite';
```